packages/components/pie-modal/src/index.ts

```diff
@@ -211,6 +218,31 @@ export class PieModal extends RtlMixin(LitElement) implements ModalProps {
         `;
     }
 
+    /**
+     * Render leadingAction button depending on prop availability.
+     *
+     * 1. If the prop `leadingAction` is not provided, the button is not rendered.
+     * 2. If the prop `leadingAction` is provided but any of the optional properties
+     * are not provided, they fall back to their default values.
+     *
+     * @param action
+     * @private
+     */
+    private renderLeadingAction (action: { text: string; variant?: string; ariaLabel?: string }): TemplateResult {
```

Comment:
Can we just access `this.leadingAction` rather than passing it into the render function? It's a class method so I would have thought it's in scope

Reply:
Fixed.